File: gpao-tex-ui/source/main/src/app/services/order/models/ordre-fabrication-value.ts
/* tslint:disable */
/* eslint-disable */
import { DetailOfValue } from '../models/detail-of-value';
export interface OrdreFabricationValue {
  chaineId?: number;
  dateDebut?: string;
  dateFin?: string;
  detailOfValues?: Array<DetailOfValue>;
  id?: number;
  informationsStock?: string;
  priorite?: string;
  produitId?: number;
  qtColisage?: number;
  qtCoupe?: number;
  qtEngagement?: number;
  qtExpedition?: number;
  qtFinition?: number;
  qtSortie?: number;
  qteEntreeAssemblage?: number;
  qteEntreeFinition?: number;
  qteEntreePreparation?: number;
  qtePriseCoupe?: number;
  qtePriseProd?: number;
  qteRecuCoupe?: number;
  qteRecueProd?: number;
  qteSortieAssemblage?: number;
  qteSortiePreparation?: number;
  quantite?: number;
  status?: string;
}

File: gpao-tex-core/production/src/main/java/com/gpro/consulting/production/dto/OrdreFabricationValue.java
package com.gpro.consulting.production.dto;

import java.util.Calendar;
import java.util.Set;

public record OrdreFabricationValue(
        Long id,
        Long chaineId,
        Set<DetailOfValue> detailOfValues,
        Integer quantite,
        Long produitId,
        Calendar dateDebut,
        Calendar dateFin,
        String priorite,
        String status,
        String informationsStock,
        Integer qtCoupe,
        Integer qtEngagement,
        Integer qtSortie,
        Integer qtFinition,
        Integer qtColisage,
        Integer qtExpedition,
        Integer qtePriseProd,
        Integer qteRecueProd,
        Integer qtePriseCoupe,
        Integer qteRecuCoupe,
        Long qteEntreeAssemblage,
        Long qteSortieAssemblage,
        Long qteEntreeFinition,
        Long qteEntreePreparation,
        Long qteSortiePreparation
) {
}
